### PR TITLE
update Kubernetes readme

### PR DESCRIPTION
**Pull request by ellis-elastic**
improved the wording of the link to Kubernetes documentation

Type of change

- Enhancement

## What does this PR do?

improved the wording of the link to Kubernetes documentation

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Review by ChrsMark — APPROVED** [review]

File: packages/kubernetes/_dev/build/docs/README.md [hunk 1]
```diff
@@ -42,8 +42,10 @@ All datasets with the `state_` prefix require `hosts` field pointing to `kube-st
 service within the cluster. As the service provides cluster-wide metrics, there's no need to fetch them per node,
 hence the recommendation is to run these datasets as part of an `Agent Deployment` with one only replica.
 
-Note: Kube-state-metrics is not deployed by default in Kubernetes. For these cases the instructions for its
-deployment are available [here](https://github.com/kubernetes/kube-state-metrics#kubernetes-deployment). 
+<DocCallOut title="Note">
+Kube-state-metrics is not deployed by default in Kubernetes. For these cases, see Kubernetes's instructions on [Kubernetes Deployment](https://github.com/kubernetes/kube-state-metrics#kubernetes-deployment). 
+</DocCallOut>
```
Comment (bmorelli25) [inline packages/kubernetes/_dev/build/docs/README.md:47]: This doc callout is specific to our new documentation build. It won't work in raw markdown. Can you remove this please?

**Review by bmorelli25 — CHANGES_REQUESTED** [review]

File: packages/kubernetes/_dev/build/docs/README.md (hunk 1 above)
Comment (ellis-elastic) [reply packages/kubernetes/_dev/build/docs/README.md]: Hi. I've done that, and pushed it up.

File: packages/kubernetes/docs/README.md [hunk 2]
```diff
@@ -42,8 +42,10 @@ All datasets with the `state_` prefix require `hosts` field pointing to `kube-st
 service within the cluster. As the service provides cluster-wide metrics, there's no need to fetch them per node,
 hence the recommendation is to run these datasets as part of an `Agent Deployment` with one only replica.
 
-Note: Kube-state-metrics is not deployed by default in Kubernetes. For these cases the instructions for its
-deployment are available [here](https://github.com/kubernetes/kube-state-metrics#kubernetes-deployment). 
+<DocCallOut title="Note">
+Kube-state-metrics is not deployed by default in Kubernetes. For these cases, see Kubernetes's instructions on [Kubernetes Deployment](https://github.com/kubernetes/kube-state-metrics#kubernetes-deployment). 
+</DocCallOut>
```
[the remark this answers is not in the document]
Comment (ellis-elastic) [reply packages/kubernetes/docs/README.md]: my mistake. I've rerun build

[the remark this answers is not in the document]
Comment (bmorelli25) [reply packages/kubernetes/docs/README.md]: Thanks!

**Pull request merged**